recipe/meta.yaml

```diff
@@ -59,11 +59,8 @@ requirements:
     # Pin all LLVM and clang packages to the latest version with each build.
     # This ensures that HOOMD can find the corresponding header files which
     # conda installs to version specific directories.
```

Comment:
This seems to be incorrect now as we are defaulting to an older version of LLVM.